src/running-the-course/translations.md

```diff
@@ -16,8 +16,9 @@ volunteers:
 - [Korean][ko] by [@keispace], [@jiyongp], [@jooyunghan], and [@namhyung].
 - [Spanish][es] by [@deavid].
 - [Ukrainian][uk] by [@git-user-cpp], [@yaremam] and [@reta].
-
-Use the language picker in the top-right corner to switch between languages.
+- [Farsi][fa] by [@DannyRavi], [@javad-jafari], [@Alix1383], [@moaminsharifi] ,
```

Comment:
Hey folks, thanks for linking the translation!

Can you restore the "Use the language picker..." sentence to be in its own paragraph?

Thanks!